### PR TITLE
Update box ~> 0.6.1

### LINKED ISSUE #31: Update vagrant-hashistack up to 0.6.X
## What is the issue?
- Outdated vagrant-hahistack.
- Intentions fix

## Suggestion(s)/solution(s) [Optional]
- Update vagrant-hahistack ~> 0.6.1

## Checklist (after created issue)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### PR DESCRIPTION
**Pull request by zhenik**
## Closes/fixes/resolves issue(s)?
- Closes #31 
## What was added/changed/fixed?
- version update
- intention fix
## Others [Optional]

I have removed directory `template_example`, prepare to next sync with `template`

## Checklist (after created PR)
- [x] Added reviewers
- [x] Added assignee(s)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### CODE REVIEW

**Review by claesgill — APPROVED** [review]
LGTM :rocket:

File: example/providers.tf [hunk 1]
```diff
@@ -9,3 +9,19 @@ provider "nomad" {
   # Add a secret_id if ACLs are enabled in nomad
   secret_id = var.nomad_acl ? data.vault_generic_secret.nomad_secret_id[0].data.secret_id : null
 }
+
+provider "vault" {
+  address = "http://127.0.0.1:8200"
+  token = "master"
```
Comment (claesgill) [inline example/providers.tf]: Could these be env variables?

Comment (zhenik) [reply example/providers.tf]: In theory - yes. But it is an example, so I would prefer to have it explicit

Comment (claesgill) [reply example/providers.tf]: I agree. Good point!

File: dev/ansible/00_create_intention.yml [hunk 2]
```diff
@@ -17,17 +19,24 @@
     CONSUL_HTTP_TOKEN: "{{ consul_token }}"
 
 #*************************************************************************************************#
-# hive-metastore => postgres (allow),hive-metastore => minio (allow)
+# hive-metastore => postgres (allow),
+#  hive-metastore => minio (allow)
```
Comment (pdmthorsrud) [inline dev/ansible/00_create_intention.yml:23]:
```suggestion
# hive-metastore => minio (allow)
```

File: example/providers.tf [hunk 3]
```diff
@@ -18,10 +18,10 @@ provider "vault" {
 terraform {
   required_providers {
     vault = {
-      version = ">=2.14.0"
+      version = ">=2.13.0"
     }
     nomad = {
-      version = ">=1.4.9"
+      version = ">=1.4.0"
```
Comment (pdmthorsrud) [inline example/providers.tf:24]: Why are we going backwards?

Comment (zhenik) [reply example/providers.tf:24]: sorry, bad copy-paste

**Review by pdmthorsrud — DISMISSED** [review]
LGTM

**Review by claesgill — APPROVED** [review]

**Review by pdmthorsrud — APPROVED** [review]

**Review by claesgill — APPROVED** [review]

**Pull request merged**